### DISCUSSION TITLE
Relay Pagination - types with description

### DISCUSSION THREAD

**Discussion by ClaudenirFreitas** [Netflix/dgs-framework#1127] [Ideas]
Hey all,

I've been using [Relay Pagination](https://netflix.github.io/dgs/advanced/relay-pagination/) that generates the relay types dynamically e.g:

```
type MessageConnection {
    edges: [MessageEdge]
    pageInfo: PageInfo
}

type MessageEdge {
    node: Message
    cursor: String
}

type PageInfo {
    hasPreviousPage: Boolean!
    hasNextPage: Boolean!
    startCursor: String
    endCursor: String
}
```

As you can see, these types don't have `description`. 
My use case is: I need to run a [graphql-schema-linter](https://github.com/cjoudrey/graphql-schema-linter) before to publish the schema to my GraphQL Schema Registry. Without these descriptions, I've been receiving the lint errors like [fields-have-descriptions](https://github.com/cjoudrey/graphql-schema-linter#fields-have-descriptions).

I already created [this branch](https://github.com/Netflix/dgs-framework/compare/master...ClaudenirFreitas:dgs-framework:feature/add-description-for-pagination-types) that adds the descriptions and it's working as expected. The output is:

```
"MessageConnection"
type MessageConnection {
  "Field edges"
  edges: [MessageEdge]
  "Field pageInfo"
  pageInfo: PageInfo
}

"MessageEdge"
type MessageEdge {
    "Field node"
    node: Message
    "Field cursor"
    cursor: String
}

"PageInfo"
type PageInfo {
    "Field hasPreviousPage"
    hasPreviousPage: Boolean!
    "Field hasNextPage"
    hasNextPage: Boolean!
    "Field startCursor"
    startCursor: String
    "Field endCursor"
    endCursor: String
}
```

I really appreciate your feedback.

Thanks in advance.

**Comment by srinivasankavitha** [comment]
Sounds good, and we just merged your PR contributions for this. Thanks for the PR!

**Reply by ClaudenirFreitas** [reply]
Cool, thanks!
